lambdas/inventory/handler.py

```diff
@@ -0,0 +1,81 @@
+import json
```

Comment:
Tell me if I am wrong, but quickly looking at this I feel this makes some assumptions about how the inventory data is stored. Are we documenting these constraints somewhere?